opal/mca/btl/vader/btl_vader_fbox.h

```diff
@@ -9,7 +9,7 @@
  * $HEADER$
  */
 
-#if !defined(MCA_BTL_VADER_FBOX_H)
+#if !defined(MCA_BTL_VADER_FBOX_H) && OPAL_BTL_VADER_FBOX_SUPPORT
```

Comment:
A minor nit: `OPAL_BTL_VADER_FBOX_SUPPORT` won't be defined until `opal_config.h` was previously included.  If `btl_vader_fbox.h` is the first file included, then that macro won't be defined.

I realize that our coding guidelines say "always include `opal_config.h` first!", but sometimes people don't/forget to do that.

Do you want to put an `#include "opal_config.h"` here at the top of `btl_vader_fbox.h` to cover all the bases / be defensive?

Reply:
You'll get a compiler error if you include fbox.h before opal_config.h, because we error on undefined preprocessor define behavior.  So I don't think it's worth being over defensive here.